source/deployment/sso-saml-adfs-msws2016.rst

```diff
@@ -165,6 +165,10 @@ Configure SAML Sign-in for Mattermost
 
 	.. image:: ../../source/images/SSO-SAML-ADFS_configure-saml_001.png
 
+Alternatively, you can create a metadata URL by appending "FederationMetadata/2007-06/FederationMetadata.xml" to the root URL of the ADFS server, for example: ``https://<adfs.domain.com>/federationmetadata/2007-06/FederationMetadata.xml>``. 
+
+Then paste this URL in the **Identity Provider Metadata URL** field, then select **Get SAML Metadata from IdP**.
```

Comment:
Same comment as for https://github.com/mattermost/docs/pull/3341 : "Maybe add another phrase that gives the rationale for the alternate way of entering the metadata Url and clicking the button - something along the lines of: "clicking this button populates the SAML SSO URL and Identity Provider Issue URL and downloads the Idp certificate from the ADFS server - you do not have to populate these fields manually anymore.".